common/v2/components/TransactionFlow/displays/FromToAccount.tsx

```diff
@@ -55,7 +59,7 @@ const LabelWrapper = styled.div`
 `;
 
 export default function FromToAccount(props: Props) {
-  const { fromAccount, toAccount } = props;
+  const { fromAddressAndLabel, toAddressAndLabel } = props;
```

Comment:
Is there a reason not to destructure the prop during the argument declaration?
```
function FromToAccount({ from, to }: Props) {}
```

It removes a LOC, but mainly it makes it really simple to know what props are used (you don't need to search for a `props.`)